data/persons.json

```diff
@@ -0,0 +1 @@
+[{"id":596,"age":123,"name":"steven","list_rental":[],"classroom":null,"parent_permission":true,"type":"student"}]
```

Comment:
- [ ] Kindly make sure that the data for  rentals is saved in its own file, based on the requirements all the data: books, people, and rentals should have separate  JSON files  👍🏿